Tamil/Thirukkural_Learning/Chapter38_fate.md

```diff
@@ -0,0 +1,18 @@
+**குறள் 371**
+
+ஆகூழால் தோன்றும் அசைவின்மை கைப்பொருள்
+போகூழால் தோன்று மடி
+
+**Explanation**
+Perseverance comes from a prosperous fate, and idleness from an adverse fate
+
+**குறள் 374**
+
+இருவே றுலகத் தியற்கை திருவேறு
+தெள்ளிய ராதலும் வேறு
+
+**Explanation**
+
+There are (through fate) two different natures in the world, hence the difference (observable in men) in (their acquisition of) wealth, and in their attainment of knowledge
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.